frontend/components/inbox_statistics/AgentResponseTimeChart.vue

```diff
@@ -40,7 +40,8 @@ export default {
     }
   }),
   async mounted() {
-    const response = await axios.get("/api" + window.location.pathname + "/agent/response/avg");
+    const inboxId = this.$route.params.inboxId
+    const response = await axios.get(`/api/inboxes/${inboxId}/agent/response/avg`);
```

Comment:
This requests returns a 404 error on localhost (docker) with user examplecoordinator

Reply:
Fixed.